src/test/resources/projects/react/reference/react-app/README.md

```diff
@@ -1,52 +1,38 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# React App
 
-## Available Scripts
+This project was bootstrapped with [`create-react-app`](https://github.com/facebook/create-react-app).
 
-In the project directory, you can run:
+This application is built to consume the AEM model of a site. It will automatically generate the layout using the helper components from the [`@adobe/cq-react-editable-components`](https://www.npmjs.com/package/@adobe/cq-react-editable-components) package.
+
+## Scripts
+
+In the project directory, you can run the following commands:
 
 ### `npm start`
 
-Runs the app in the development mode by proxying the JSON model from a local AEM instance running at http://localhost:4502.
+Runs the app in development mode by proxying the JSON model from a local AEM instance running at http://localhost:4502. This assumes that the entire project has been deployed to AEM at least once (`mvn clean install -PautoInstallPackage`).
 
-This assumes that the entire project has been deployed to AEM at least once.
+Your app can be viewed on http://localhost:4502/content/test-spa-react-project/en/home.html and the page will reload if you make edits.
```

Comment:
I am not sure, but I was of the opinion that a Front-End Developer would use this command - `npm start` to do local development where he would use the localhost instance started by this command instead of the AEM instance.
i.e. FED would play his code on http://localhost:3000

The Model JSON might come from AEM (in which case the project needs to be uploaded on AEM once), or I remember we had the option for Proxy Json and Mock Json as well. Is that no longer the case ?

Reply:
You're right, this should be http://localhost:3000. I'll fix this.

Once https://github.com/adobe/aem-spa-project-archetype/pull/118 is merged, we'll be able to remove this line completely.